drivers/mac/controller.m

```diff
@@ -113,56 +111,58 @@ + (void) listen:(NSDictionary *)in return:(NSString *)returnID {
     // joysticks + dpad
     controller.profile.dpad.valueChangedHandler = ^(GCControllerDirectionPad *dpad, float x, float y) {
         // Directional pad has been changed
-        [Controller emitDpad:ID Input:0 X:x Y:y];
+        [Controller emitDpad:ID Input:DirectionalPad X:x Y:y];
```

Comment:
I think you can remove all these comments about what it does since the code self explain.